src/js/components/Button/__tests__/Button-test.js

```diff
@@ -26,6 +26,23 @@ describe('Button', () => {
     expect(results).toHaveNoViolations();
   });
 
+  test('passes through the aria-label prop', async () => {
+    const TEST_LABEL = 'Test Label';
+    const { container, getByText } = render(
+      <Grommet>
+        <Button aria-label={TEST_LABEL} label="Test" onClick={() => {}} />
+      </Grommet>,
+    );
+
+    const button = container.querySelector('button');
+    expect(button).toBeTruthy();
+    expect(button.getAttribute('aria-label')).toEqual('Test Label');
+
+    fireEvent.click(getByText('Test'));
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
```

Comment:
Where are the snapshots created for this test that are indicating that `aria-label` is applied as expected?

Reply:
There are no snapshots, I'm just manually asserting that the attribute is correctly set. If you find a snapshot valuable, I'm happy to add it.

Reply:
@ShimiSun I've added a snapshot that looks at both `a11yTitle` and `aria-label` props.